components/benefit_sponsors/app/models/benefit_sponsors/benefit_sponsorships/benefit_sponsorship_director.rb

```diff
@@ -12,6 +12,7 @@ def process(benefit_sponsorships, event)
       business_policy_name = policy_name(event)
 
       benefit_sponsorships.no_timeout.each do |benefit_sponsorship|
+        @logger.info "Event (#{event}) process started for Employer #{benefit_sponsorship.legal_name}(#{benefit_sponsorship.fein})"
```

Comment:
do we have acces to the Safe Navigation Operator ? it would be good to use it here for `benefit_sponsorship&.legal_name`

Reply:
also, can you move this inside `begin` block just to be on safer side.